firebase_extension_examples/resize_images/src/components/images.tsx

```diff
@@ -0,0 +1,100 @@
+import { builder } from "@invertase/image-processing-api";
+
+/** Add project config */
+const PROJECT_ID = "";
+const PROJECT_LOCATION = "$";
+
+/** List names of images from a GCS source */
+const imageSources = [
+  {
+    id: 1,
+    title: "Mountains",
+    //image: "travelling/pexels-amine-m'siouri-2108813.jpg",
```

Comment:
Is this line/object needed?

Reply:
Yeah, it's technically a placeholder. As the image path needs to be hardcoded to match an image in your storage bucket